packages/gn.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Gn < Package
+  description 'GN is a meta-build system that generates build files for Ninja.'
+  homepage 'https://gn.googlesource.com/gn/'
+  version 'dca877f'
+  source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT'
+  source_sha256 'b58d799b16d20abf92369fe0749c73f7398996f0afa9933517051778a8bb16c3'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+
```

Comment:
Missing `depends_on 'python27'`.

Reply:
Missing `depends_on 'meson' => :build`.

Reply:
These are installed on my config. So I didn't note these missings. Is there a tool to get these infos ?

Reply:
You can find out which package you don't need to include with: `grep <package> /path/to/cloned/chromebrew/tools/core_packages.txt`.  You can find out which packages you need simply by examining all the commands you are using.  You need `python27` since you are using the `python` command.  You need `meson` since you are using the `ninja` command and the meson build system.